website_multi_theme/tests/test_assets.py

```diff
@@ -40,9 +19,9 @@ def test_localhost(self):
             """//head/link[contains(@href,
                'website_multi_theme.auto_assets_website')]"""))
 
-    def test_127_0_0_1(self):
-        """Check 127.0.0.1 downloads its default assets."""
-        response = self.url_open("http://127.0.0.1:%d" % PORT, timeout=30)
+    def test_0_0_0_0(self):
+        """Check 0.0.0.0 downloads its default assets."""
+        response = self.url_open("http://0.0.0.0:%d" % PORT, timeout=30)
```

Comment:
Wow, TIL that you can use `0.0.0.0` to call to localhost!

Reply:
@yelizariev what was your reason to change this? currently facing travis error on these tests in v10. both 127.0.0.1 and 0.0.0.0 are failing the tests. localhost is working just fine.

Reply:
It allows using exising ``website`` records. Before the update new website was created
```

            ip = env["website"].create({
                "name": "127.0.0.1",
                "domain": "127.0.0.1",
                "multi_theme_id": False,
            })
```